.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
Let's drop this file from this PR. We only actually need the template on main and it doesn't have to exist on release-1.2